core/utils/http_allowed_ips.go

```diff
@@ -70,7 +70,7 @@ func restrictedDialContext(ctx context.Context, network, address string) (net.Co
 
 		if isRestrictedIP(a.IP) {
 			defer logger.ErrorIfCalling(con.Close)
-			return nil, fmt.Errorf("disallowed IP %s. Connections to local/private and multicast networks are disabled by default for security reasons. If you really want to allow this, consider using the httpgetwithunrestrictednetworkaccess or httppostwithunrestrictednetworkaccess adapter instead", a.IP.String())
+			return nil, fmt.Errorf("disallowed IP %s. Connections to local/private and multicast networks are disabled by default for security reasons. If you really want to allow this, consider setting the allowunrestrictednetworkaccess option on the task to true", a.IP.String())
```

Comment:
For legacy job pipelines the old error message still applies. We may want to clarify, like this:

"For legacy (v1) job pipelines, consider using the httpgetwithunrestrictednetworkaccess or httppostwithunrestrictednetworkaccess adapter. For v2 job pipelines, you will need to set the allowunrestrictednetworkaccess option on the task to true"